### PR TITLE
Fix updating gene panel name displayed on report page

### LINKED ISSUE #296: Issues with customizing report through gene panel name
In the gene panel customize:

chanjo2_customize

The panel can be updated using a gene panel name. This was something our geneticists instinctively immediately tried.

I am unsure whether it works as intended though. Updating the name triggers a page load, but:

- The gene panel name disappeared (in my case at least)
- Only one gene was shown (same as before the update) despite the panel having 28 genes

I used both the "Full name" and the "Panel ID" of the panel when trying this out.

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #296 

### How to test:
Locally, from demo report: http://localhost:8000/report/demo
- Use Customize form to change displayed gene panel
- Gene panel should be modified

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally 🆗 

image

File: src/chanjo2/templates/report.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h2 class="accordion-header" id="flush-headingOne">
 
 				<div class="col-6">
 				  <label class="form-label">Gene panel name to display</label>
-				  <input class="form-control" id="panel_name" type="text" placeholder="Skeletal dysplasia 3.2" value="{{ extras.panel_name or '' }}">
+				  <input class="form-control" id="panel_name" name="panel_name" type="text" placeholder="Skeletal dysplasia 3.2" value="{{ extras.panel_name or '' }}">
```
Comment (northwestwitch) [inline src/chanjo2/templates/report.html:44]: This is the fix

**Comment by northwestwitch** [comment]
I wrote a fix to the panel renaming error. You can review if you want @Jakob37

**Review by Jakob37 — APPROVED** [review]
Looks good! Tested locally, behaved as expected with the name updated:

updated_name

**Comment by northwestwitch** [comment]
Thanks! Merging

**Pull request merged**